seeders/20200121095153-demo-user.js

```diff
@@ -4,15 +4,15 @@ const up = (queryInterface) => queryInterface.bulkInsert('Users', [{
   name: 'Dummy',
   gender: 'Male',
   email: 'dummy@email.rw',
-  username: 'MrDummy',
+  username: 'Dummy',
   password: Hasher.hashPassword('123456789'),
   birthdate: new Date(),
   preferredLanguage: 'French',
   preferredCurrency: 'Euro',
-  location: 'Paris',
-  role: 'Broker',
+  locationId: 3,
+  role: 'Requester',
   department: 'Cleaner',
-  lineManager: 'MrNobody',
+  lineManager: 'Runyamahe',
```

Comment:
Is a line manager a string or a real user of manager role? work on this with everyone else who is using the same settings.

Reply:
Here the line manager is the username of a user with a role of manager